packages/builder/src/components/common/NavItem.svelte

```diff
@@ -71,6 +73,10 @@
   on:dragstart
   on:dragover
   on:drop
+  on:mouseover
+  on:mouseout
+  on:focus
+  on:blur
```

Comment:
As mentioned, we should update to use enter and leave instead. I don't see a usage for `focus` or `blur` here - do we need them? Otherwise it would be better to remove an additional 2 event handlers on every nav item.

Reply:
Originally added this to silence a linting error complaining about the lack of them when specifying mouseover and mouseout, I assumed if they were unused they wouldn't generate handlers, but perhaps I'm wrong. But since we're using mouseenter and mouseleave now the linter is no longer complaining, so I removed this.